client/src/views/adminDashboard/components/calendar/Calendar.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Calendar() {
+
+  return (
+    <div>
+      Calendar
+    </div>
```

Comment:
I don't think you can navigate to this file, so I'm not sure why you'd want to check it in.

Reply:
YAGNI.